src/utils/apiControllers/apiPath.ts

```diff
@@ -0,0 +1,12 @@
+export const apiPath = {
+  login: 'https://ya-praktikum.tech/api/v2/auth/signin',
```

Comment:
Если создашь файл config.js (как я писал в одном из комментариев выше), то каждый из этих урлов можно будет упростить
```suggestion
  login: '/api/v2/auth/signin',
```